app/controllers/application_controller.rb

```diff
@@ -1,5 +1,9 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  
+  include ApplicationControllerImpl
+  
+  public
 
```

Comment:
Why do I have to add this `public` here? (took it from Kuina). It seems to me not required and `robocop` consider it an offense.